packages/ProgressBar/src/ProgressBar.js

```diff
@@ -18,7 +18,7 @@ const propTypes = {
   header: PropTypes.node,
 
   /** Semantic heading level of header */
-  headerLevel: PropTypes.oneOf[(1, 2, 3, 4, 5, 6)],
+  headerLevel: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
what is the change here :D 🤔 ?

Reply:
The brackets.  I don't even know how the original version was syntactically correct.

Reply:
ohh I found it, hahah was not an array before 🤦